integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -448,7 +448,7 @@ shortCircuit:
   policy: uuid
   # volumeType controls the type of shortCircuit volume.
   # It can be "persistentVolumeClaim" or "hostPath"
-  volumeType: persistentVolumeClaim
+  volumeType: hostPath
```

Comment:
There are two orthogonal topics and I'd like to discuss separately.

**Topic 1: How to effectively use `hostPath` without too much user trouble**

We've had this struggle on worker tiered storage (that supports hostPath too). Having a initContainer which requests root permission (without users knowing) is somewhat tricky because it brings about a chain of user questions:
1. Why does Alluxio want root permission (and hence rejected by our admin or `PodSecurityPolicy`)?
2. Then what do I do if I want to use Alluxio without root permission? -- We then had to provide the instructions in our docs anyway.

Let's just talk about the default config here, as we want minimal user perception and trouble. We want users to just start Alluxio with a minimal setup and try, and succeed without tweak for 2 hours.

Then supporting `emptyDir` is necessary for such test env, and that doesn't take extra permission mgmt. So do you want the default to be `emptyDir`? My answer would be -- if you look around and see other pods using `emptyDir` as the default volume type, that suggests we guess the user wants the default helm chart to be a test setup. Then use `emptyDir` here as the default, to play along. Otherwise if you feel all other code favor a production/serious deployment like PVC, that we used to assume admins know what they are doing when they see "PVC", play along too.

I've been away from the helm chart for a while and can't remember what we favor here (test or prod). So i'll leave it up to you and your trusted community users for which one to favor -- but favor that consistently.

**Topic 2: Does PVC work at all?**

In short, this domain socket volume is the same as other worker tiered storage volumes. So if we support tiered store to be PVC, that suggests domain socket as PVC works.

You want this volume to be node-local and next to the Pod. Then we use DaemonSet to make sure each worker claims its exclusive local volume. If tiered storage works with PVC, then domain socket would the same way. If the PVC mounts to a PV of `hostPath` or `local`, then each worker will go to its local path even though the PV/PVC types are `ReadWriteOnce`.